app/uk/gov/hmrc/hec/controllers/TaxCheckController.scala

```diff
@@ -36,11 +38,25 @@ class TaxCheckController @Inject() (
   taxCheckService: TaxCheckService,
   authenticateGG: GGAuthenticateAction,
   authenticatedGGOrStride: GGOrStrideAuthenticateAction,
+  config: Configuration,
+  auth: BackendAuthComponents,
   cc: ControllerComponents
 )(implicit ec: ExecutionContext)
     extends BackendController(cc)
     with Logging {
 
+  val internalAuthEnabled: Boolean = config.get[Boolean]("internal-auth.enabled")
+
+  val permission: Predicate.Permission = Predicate.Permission(
+    resource = Resource(
+      resourceType = ResourceType("hec"),
+      //while creating token for internal auth, resource allocation = /hec/match-tax-check was giving error stating no forward slash allowed.
```

Comment:
that's weird - what command did you use to try to generate the token with a slash in it?

Reply:
`curl -X POST http://localhost:8470/test-only/token -d '{"token": "123457","principal": "hec","permissions": [{"resourceType": "hec","resourceLocation": "/hec/match-tax-check","actions": ["READ"]}]}' -H 'Content-Type:application/json'`

Reply:
yeah I can see it's complaining about the leading slash - could we change it to `hec/match-tax-check` then please? I'll raise a PR to get it changed in internal-auth-config

Reply:
yeah sure

Reply:
[updated](https://github.com/hmrc/hec/pull/55/commits/3dd88d65dfd4c7df63b233da1cc9608d0ee25a1d)